src/index.js

```diff
@@ -3,40 +3,39 @@ import mergeBases from './merge-bases.js'
 import linkIssues from './link-issues.js'
 import autoMerge from './auto-merge.js'
 import trackVersions from './track-versions.js'
+import corgiTagWatcher from './corgi-tag-watcher.js'
 import slackStuff from './slack-stuff.js'
-import addonSettings from 'probot-addon-settings'
 
 const { IGNORE_FOR_TESTING } = process.env
 
-export default (robot) => {
-  robot.events.setMaxListeners(100) // Since we use multiple plugins
+export default (robot, { getRouter }) => {
+  // robot.events.setMaxListeners(100) // Since we use multiple plugins
```

Comment:
I'm not 100% sure, I'll need to ask @Phil. It was not working, but perhaps it just needs a change to work with the new version of Probot.

Reply:
I'd say remove it. I added it because an earlier version of Probot used the default number of event listeners (10) and staxly listens to more than 10 GitHub events. The failure would have occurred on startup and since it doesn't that code can be removed :broom: